forms-flow-api/src/formsflow_api/models/form_process_mapper.py

```diff
@@ -182,6 +182,36 @@ def get_latest_form_mapper_ids(cls):
             .all()
         )
 
+    @classmethod
+    def add_sort_filter(cls, query, sort_by, sort_order):
+        """Adding sortBy and sortOrder."""
+        order = []
+        if sort_by and sort_order:
+            for sort_by_att, sort_order_attr in zip(sort_by, sort_order):
+                name, value = validate_sort_order_and_order_by(
+                    sort_order=sort_order_attr, order_by=sort_by_att
+                )
+                if name and value:
+                    order.append(text(f"form_process_mapper.{name} {value}"))
+
+            query = query.order_by(*order)
+        return query
+
+    @classmethod
+    def add_search_filter(cls, query, search):
+        """Adding search filter in query."""
+        if search:
+            filters = []
+            for term in search:
```

Comment:
Wouldn't `for term in search:` split each of the letters ?
i.e "this form" would give ['t', 'h', 'i', 's', 'f', 'o', 'r', 'm']

If yes, Instead we should be using `for term in search.split()`